src/main/java/org/graylog/integrations/aws/resources/AWSResource.java

```diff
@@ -118,4 +124,43 @@ public Response kinesisHealthCheck(@ApiParam(name = "JSON body", required = true
         KinesisHealthCheckResponse response = kinesisService.healthCheck(heathCheckRequest);
         return Response.accepted().entity(response).build();
     }
+
+    /**
+     * Save the new AWS CloudWatch integration.
+     *
+     *  curl 'http://admin:123123123@localhost:9000/api/plugins/org.graylog.integrations/aws/kinesis/save' \
+     *  -v \
+     *  -X POST \
+     *  -H 'X-Requested-By: just-a-test' \
+     *  -H 'Content-Type: application/json' \
+     *  -H 'Accept: application/json' \
+     *  --compressed \
+     *  --data-binary '{
+     *      "aws_access_key": "",
+     *      "aws_secret_key": "",
+     *      "region": "us-east-1",
+     *      "name": "New Flow Logs",
+     *      "description": "Some flow logs.",
+     *      "aws_input_type": "KINESIS_FLOW_LOGS",
+     *      "stream_name": "flow-logs",
+     *      "batch_size": 10000,
+     *      "assume_role_arn": "",
+     *      "global": false,
+     *      "enable_throttling": false
+     * }'
+     */
+    @POST
+    @Timed
+    @Path("/save")
```

Comment:
How about using `/aws/inputs` as path for saving inputs? That's a bit more specific than `/aws/save`.